### DISCUSSION TITLE
Make booting from UEFI the default on vm creation

### DISCUSSION THREAD

**Discussion by SludgeGirl** [cockpit-project/cockpit-machines#1946] [Ideas]
What are the opinions on potentially making UEFI the default when creating vm's? Providing a drop-down on the initial creation popup that default to UEFI I think would be a nicer work flow than creating the vm stopped and then changing the boot mode. When set to UEFI providing `--boot uefi` and when set to legacy leaving the option unset to go back to the original behavior

My justification for this comes from the explanation here: https://code.opensuse.org/leap/features/issue/39
I felt it raised some good enough points that it'd be worth checking with you

I'm happy to work on this if it's something we think is a good idea!

**Comment by jelly** [comment]
We had this requested as an https://github.com/cockpit-project/cockpit-machines/issues/1912 a while ago, ideally virt-install would switch to an `uefi` default for us but there hasn't been recent activity on the relevant issue https://github.com/virt-manager/virt-manager/issues/757
